### PR TITLE
Version number alignement

### PR DESCRIPTION
**Pull request by m1kah**
Version numbers were a bit hard to read because they did
continue on next row without padding.

Improved readability by aligning each row of version numbers
with same padding. Also aligning version numbers with
variable width so that they are printed in columns.

### CODE REVIEW

**Comment by holmbech** [comment]
Will look at it this weekend and getting it merged in

File: src/main/groovy/com/nordea/devtool/Devtool.groovy [hunk 1]
```diff
@@ -434,32 +434,102 @@ class Devtool {
...
+        // use some reasonable minimum width. 25 is used padding.
+        if (columns < 60) {
+            columns = 80
+        }
+        return columns
+    }
+
+    int bashColumns() {
+        try {
+            ProcessBuilder processBuilder = new ProcessBuilder("tput", "cols");
```
Comment (holmbech) [inline src/main/groovy/com/nordea/devtool/Devtool.groovy:503]: the "tput" is not part of the standard windows cmd. How are you running it?

Comment (m1kah) [reply src/main/groovy/com/nordea/devtool/Devtool.groovy:503]: "tput" is part of Git Bash so it provides number of columns when running Git Bash in Windows or any Linux shell.

Comment (holmbech) [reply src/main/groovy/com/nordea/devtool/Devtool.groovy:503]: ok. And now that I have actually looked at it again (with obviously more fresh eyes) I can see that you handle both a unix prompt and standard Windows prompt. :-)
I'll merge it it now.
I'll think I will have a go at a bit of refactoring and also create a setup for easier local testing without have e nexus server or the like up and running.

**Pull request merged**